### ISSUE TITLE
Single placement of AR Interactable when using AR Placement Interactable

### ISSUE THREAD

**Issue by ROBYER1** [Unity-Technologies/XR-Interaction-Toolkit-Examples#44]
Using AR Placement Interactable, is there a means to add just one single AR Interactable prefab in the scene and not keeping adding more?

Primary use case for this is for apps where the user puts down one object in AR and wants to move/rotate it easily without additional clicks causing additions of extra prefabs into the scene through touch events registered by AR Placement Interactable.

**Comment by ACWesterberg** [comment]
Yes make an extension to the script and implement your own solution. This video shows a way of doing it https://www.youtube.com/watch?v=g2_OzmUG1N0&list=PLQMQNmwN3Fvx2d7uNxMkVOs1aUV-vxrlf&index=11&ab_channel=DilmerValecillos
